src/commands/set_command.py

```diff
@@ -0,0 +1,41 @@
+from commands.abstract_command import *
+from model.database import *
+
+
+class SetCommand(AbstractCommand):
+    def __init__(self, database, table, value, overwrite = False):
+        SetCommand.validate(str(database), str(table), value, overwrite)
+        self.table = Database(Database.get_schema_data(str(database))).get_table(str(table))
```

Comment:
Command doesn’t deal with table object.
It deals with database object.

I expect to have Database.load(database_name